content/docs/command-reference/remote/modify.md

```diff
@@ -279,33 +278,35 @@ a full guide on using Google Drive as DVC remote storage.
                       url gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
   ```
 
-- `gdrive_client_id` - **Client ID** for authentication with OAuth 2.0 when
-  using a
+- `gdrive_client_id` - Client ID for authentication with OAuth 2.0 when using a
   [custom Google Client project](/doc/user-guide/setup-google-drive-remote#using-a-custom-google-cloud-project).
   Also requires using `gdrive_client_secret`.
 
   ```dvc
-  $ dvc remote modify myremote gdrive_client_id <client ID>
+  $ dvc remote modify --local myremote gdrive_client_id <client ID>
```

Comment:
these are fine to be shared, everything for GDrive is more or less fine to share

Reply:
True, I didn't think this one through. Updated.

But I think it's safer to leave `gdrive_user_credentials_file` as --local. Even when the path itself is not THE secret info, in strict terms paths to private files are considered private as well. This is why I included other paths similarly, but in the paragraph for those I say "**may** contain sensitive user info", depending on each case.